src/snowflake/cli/plugins/nativeapp/teardown_processor.py

```diff
@@ -89,9 +97,28 @@ def drop_application(self, auto_yes: bool):
                     cc.message(f"Did not drop application object {self.app_name}.")
                     return  # The user desires to keep the app, therefore exit gracefully
 
-        # 4. All validations have passed, drop object
+        # 4. If cascade was not explicitly set, check for objects owned by the app
+        if cascade is None:
+            has_compute_pools = False  # SHOW OBJECTS OWNED BY APPLICATION
+            if has_compute_pools:
+                if interactive:
+                    cascade = typer.confirm(
+                        "There are objects owned by this app. Should they be deleted?"
```

Comment:
We should show the user what these are, otherwise the question is really difficult to answer.

Also, any kind of "should I do <something destructive>" should probably have 3 possible answers: yes, no, abort.

Reply:
Added the list of the files in question -- see PR description for a demo.

Do you think it's reasonable to assume users know to abort the script? (ie Ctrl+C)
Typer has nice out of the box support for boolean values, but by switching to yes/no/abort I will need to parse the input myself and it might get inconsistent with other boolean values. WDYT?